csrankings-p.csv

```diff
@@ -1003,7 +1003,7 @@ Prasad Jayanti,Dartmouth College,http://www.cs.dartmouth.edu/~prasad,hdn0Ln0AAAA
 Prasad Kulkarni,University of Kansas,http://www.ittc.ku.edu/~kulkarni,ETTvwdQAAAAJ
 Prasad Raghavendra,Univ. of California - Berkeley,http://freevideolectures.com/Course/3567/Efficient-Algorithms-and-Intractable-Problems,t5KSayQAAAAJ
 Prasad Tadepalli,Oregon State University,http://eecs.oregonstate.edu/~tadepall,CXAN0i0AAAAJ
-Prasad Tetali,Georgia Institute of Technology,https://people.math.gatech.edu/~tetali,NOSCHOLARPAGE
+Prasad Tetali,Carnegie Mellon University,https://tetali.github.io/,NOSCHOLARPAGE

```

Comment:
Math, not SCS. Please explain how he meets the inclusion criteria, re-opening if that's the case.